apps/.eslintrc.js

```diff
@@ -130,6 +130,7 @@ module.exports = {
       rules: {
         // We can rely on typescript types instead of prop types
         'react/prop-types': 'off',
+        '@typescript-eslint/no-unused-vars': ['error', {args: 'none'}],
```

Comment:
This matches our existing JS config for `eslint/no-unused-vars`, seemed to be automatically used previously, now requires an explicit redeclaration for TS.